packages/data/src/data-manager.ts

```diff
@@ -247,7 +247,7 @@ export class DataManager implements DataManagerInterface {
       ) {
         isBucketed = true;
       }
-      let locationMatched: boolean | RuleError = false,
+      let locationMatched: boolean | RuleError = true, // Empty experience locations list or unset Site Area means there's no restriction for the location
```

Comment:
hmm, I do not think it's ok... if `!locationProperties` this would still remain true, which is not what you want...you want it to stay true if Empty experience locations list and unset Site Area

Reply:
Makes sense, so we should not proceed if we do not have `locationProperties`